src/test/scala/edu/ie3/simona/agent/participant/EvcsAgentModelCalculationSpec.scala

```diff
@@ -904,7 +881,7 @@ class EvcsAgentModelCalculationSpec
           1
         )
       )
-      scheduler.expectMsg(CompletionMessage(5L))
+      scheduler.expectNoMessage()
```

Comment:
Is this check needed then? In line 848 you removed it. So I would opt for having it at both positions (if necessary and sensible) or delete it here as well.